### PR TITLE
Moves all calls to .all() within the */service.py files

### PR DESCRIPTION
**Pull request by metroid-samus**
This PR moves all session queries with `all.()` calls to `*/service.py` files for consistency.

### CODE REVIEW

**Comment by wssheldon** [comment]
Nice! I think another good addition here would be to fix all the return types. That way, if you try to call `.all()` on one of these service functions you will get a type error w/ `pyright`.

`list[Whatever] | None` in most cases, I believe.

File: src/dispatch/tag/scheduled.py [hunk 1]
```diff
@@ -54,7 +55,7 @@ def sync_tags(db_session: SessionLocal, project: Project) -> NoReturn:
 @scheduled_project_task
 def build_tag_models(db_session: SessionLocal, project: Project) -> NoReturn:
```
Comment (wssheldon) [inline src/dispatch/tag/scheduled.py:56]:
```suggestion
def build_tag_models(db_session: SessionLocal, project: Project) -> None:
```

File: src/dispatch/case/service.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ def get_by_name_or_raise(*, db_session, project_id: int, case_in: CaseRead) -> C
 
 def get_all(*, db_session, project_id: int) -> List[Optional[Case]]:
```
Comment (wssheldon) [inline src/dispatch/case/service.py:70]: `List` and `Optional` are both deprecated. `List` -> `list` (no import required) and `Optional` becomes `Case | None` (no import required)

recommend we use this syntax going forward for consistency.

Comment (metroid-samus) [reply src/dispatch/case/service.py:70]: Acknowledged. I've created a separate PR to handle the List/Optional/Union changes.
https://github.com/Netflix/dispatch/pull/4744

**Pull request closed without merging**